Lesson 4/olha.bezpalchuk/script.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+if [ "$EUID" -ne 0 ]
+then
+  echo 'Please run as root'
+  exit
+fi
+
+function debugMessage() {
+  echo -e '\e[32m>' $1 '\e[39m'
+}
+
+# 1. Show list of installed programs and check if nginx is in there
+debugMessage '\t===== task 1 ====='
+debugMessage 'checking for nginx...'
+if dpkg -l | grep nginx > /dev/null 2>&1;
+then
+  # 1.1 If yes, uninstall nginx and show message about deleting. Specify uninstalled version.
+  debugMessage 'nginx found'
+  VERSION="$(nginx -v)"
+  debugMessage "uninstalling nginx version ${VERSION}..."
+  apt-get -y purge nginx nginx-common
+  apt-get -y autoremove
+  debugMessage "> nginx uninstalling - done (version ${VERSION})"
+else
+  # 1.2 If no, show message about not being installed
+  debugMessage 'nginx not found'
+fi
+
+#2 Add nginx remote repository (using documentation) and install nginx 1.14.2
+debugMessage '\t===== task 2 ====='
+debugMessage 'installing nginx 1.14.2...'
+# install the prerequisites
+apt-get -y install curl gnupg2 ca-certificates lsb-release
```

Comment:
Package gnupg2 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source